src/tools/tidy/src/deps.rs

```diff
@@ -47,6 +47,7 @@ const EXCEPTIONS: &[(&str, &str)] = &[
     ("instant", "BSD-3-Clause"),                             // rustc_driver/tracing-subscriber/parking_lot
     ("mdbook", "MPL-2.0"),                                   // mdbook
     ("openssl", "Apache-2.0"),                               // opt-dist
+    ("option-ext", "MPL-2.0"),                               // cargo-miri (via `directories`)
```

Comment:
@rust-lang/compiler I am assuming that it is okay to add another MPL-2.0 dependency to cargo-miri.